vm/src/obj/objbool.rs

```diff
@@ -60,6 +65,26 @@ pub fn get_value(obj: &PyObjectRef) -> bool {
     }
 }
 
+fn bool_to_int(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(vm, args, required = [(obj, Some(vm.ctx.int_type()))]);
+
+    if let PyObjectPayload::Integer { value } = &obj.borrow().payload {
```

Comment:
You can use objint::get_value to perform this logic for you. Please use it, to make the code smaller.

Reply:
Done